evap/results/tests/test_views.py

```diff
@@ -152,6 +153,58 @@ def test_wrong_state(self):
         url = '/results/semester/%s/course/%s' % (self.semester.id, course.id)
         self.get_assert_403(url, 'student')
 
+
+class TestResultsSemesterCourseDetailViewFewVoters(ViewTest):
+    @classmethod
+    def setUpTestData(cls):
+        semester = mommy.make(Semester, id=2)
+        mommy.make(UserProfile, username='evap', groups=[Group.objects.get(name='Staff')], email="evap@institution.example.com")
```

Comment:
Is there a semantic difference between the user 'staff' we often have in tests and this 'evap' user that appears a lot less often? Also, do we need the fixed email address?

Reply:
the name is not really important. we need a fixed email address to make it an internal user.

Reply:
I would suggest using 'staff' everywhere then. Currently, we have 45 times 'staff' and 6 times 'evap'.